specs/commandline/sign.md

```diff
@@ -31,9 +31,11 @@ Flags:
   -d,  --debug                      debug mode
   -e,  --expiry duration            optional expiry that provides a "best by use" time for the artifact. The duration is specified in minutes(m) and/or hours(h). For example: 12h, 30m, 3h20m
   -h,  --help                       help for sign
+       --id string                  key id (required if --plugin is set)
   -k,  --key string                 signing key name, for a key previously added to notation's key list.
   -p,  --password string            password for registry operations (default to $NOTATION_PASSWORD if not specified)
        --plain-http                 registry access via plain HTTP
+       --plugin string              signing plugin name
```

Comment:
Based on our previous discussion, we should mention this flag is mutual exclusive with the `--key` flag in the description.